backend/siarnaq/api/teams/views.py

```diff
@@ -173,3 +175,25 @@ def avatar(self, request, pk=None, *, episode_id):
             titan.upload_image(avatar, profile.get_avatar_path())
 
         return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+    @action(
+        detail=False,
+        methods=["post"],
+        serializer_class=None,
+        permission_classes=(IsAuthenticated, IsEpisodeAvailable),
+    )
+    def record(self, request, pk=None, *, episode_id):
+        """Retrieve the win/loss record of a team"""
+        team = request.data["id"]
+        matches = Match.objects.filter(participants__team=team)
+
+        matches.annotate(
+            score_1=Max("participants__score", filter=Q(participants__team=team)),
+            score_2=Max("participants__score", filter=~Q(participants__team=team)),
+        )
+
+        win_count = matches.filter(score_1__gt=F("score_2")).count()
```

Comment:
I get an error that claims score_1 cant be resolved to a field. But I'm using annotate and filter almost copy paste from django documentation? [https://docs.djangoproject.com/en/4.1/topics/db/aggregation/](url)

Reply:
`.annotate` returns a new queryset so you need `matches = matches.annotate(...)`